BoxContentSDK/BoxContentSDK/OAuth2/BOXAbstractSession.m

```diff
@@ -227,7 +227,7 @@ + (NSArray *)usersInKeychain
         NSArray* keychainEntries = (__bridge_transfer NSArray*)keychainQueryResult;
         for (NSDictionary *dict in keychainEntries)
         {
-            NSObject *object = [dict objectForKey:((__bridge id)kSecAttrGeneric)];
+            NSObject *object = [dict objectForKey:((__bridge id)kSecAttrAccount)];
```

Comment:
kSecAttrGeneric is supposed to have a CFData type. On iOS, it's returning a string when we pass a string into it, but I'm not sure that's behavior we should rely on. In OS X, it always returns a string. Since we put the same value into kSecAttrAccount, and that is supposed to be a string type, I switched to using that value instead.